superset/assets/src/chart/chartAction.js

```diff
@@ -246,6 +247,16 @@ export function runQuery(formData, force = false, timeout = 60, key) {
   };
 }
 
+export const FETCH_CHART = 'FETCH_CHART';
+export function fetchChart(formData, force = false, timeout = 60, key) {
```

Comment:
can you give these more specific names, and possibly leave a comment as to how they are different? 

If I was a new dev to Superset, I would be very confused why there were two nearly identical methods that only change GET/POST. e.g., `fetchCachedChart` (or similar) and `runChartQuery` might be more clear.

Reply:
Will do.